cloudbuild.yaml

```diff
@@ -4,7 +4,7 @@ timeout: 3600s
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: 'golang:1.20.12'
+  - name: 'golang:1.22.2'
```

Comment:
So ... are we not using bazel for this? We're doing both go builds and bazel and have to keep the versions in sync? 😬

Reply:
I honestly lost track on what builds what and how